src/lib389/lib389/agreement.py

```diff
@@ -328,6 +331,13 @@ def status(self, winsync=False, just_status=False, use_json=False, binddn=None,
         # RUV entry under the suffix, then we can't get the status.  So in this case we
         # need to provide a DN and password.
         if not winsync:
+            if pwprompt:
+                host = self.get_attr_val_utf8(AGMT_HOST)
+                port = self.get_attr_val_utf8(AGMT_PORT)
+                if binddn is None:
+                    binddn = _get_arg(None, msg=f"Enter bind DN for {host}:{port}")
```

Comment:
At this point, would it be possible to display the replicated suffix ?
If there are several replicated suffixes the password for each replica may differs.